src/responseUtils.ts

```diff
@@ -5,62 +5,44 @@
 import { Logger, type ILogObj } from 'tslog';
 import { HttpResponse } from 'uWebSockets.js';
 
-export const forbidden = (res: HttpResponse, LOG: Logger<ILogObj>, e: Error) => {
+const response = (
+	res: HttpResponse,
+	LOG: Logger<ILogObj>,
+	e: Error,
+	statusCode: string,
+	msg: string | undefined = undefined
```

Comment:
Good idea to have a function (response maybe too generic) I would use a type for these fields... So you can have validation over the values

Reply:
Do you mean to have string literal types for `statusCode` and `msg`?

Reply:
yes more an enum like: https://gist.github.com/scokmen/f813c904ef79022e84ab2409574d1b45?permalink_comment_id=4581061#gistcomment-4581061
and msg could also be an enum of string/list!

Reply:
I left `msg` as a simple string since it can comes from the error and thus it has no prefixed values. 

Moreover I would like to extend the use of httpResponse everywhere, but this require more time since it is not obvious how to handle custom response codes and headers (that can comes from the contract metadata). Thus will address it in a different PR